src/components/ListservSignup/ListservSignup.css

```diff
@@ -0,0 +1,4 @@
+.seatbutton {
```

Comment:
Hmm I don't love this solution, it doesn't scale well with screen size
I'd consider re-writing the formatting with some combination of reacstraps <Row> and <Col> components

Reply:
Yea I didn't like it either. I'll take another look at it with your suggestion in mind. We should chat after the studio Sunday.